lambda/examples/error-handling.rs

```diff
@@ -0,0 +1,107 @@
+/// See https://github.com/awslabs/aws-lambda-rust-runtime for more info on Rust runtime for AWS Lambda
+use lambda::handler_fn;
+use serde::{Deserialize, Serialize};
+use serde_json::{json, Value};
+use simple_error;
+use simple_logger;
```

Comment:
```suggestion
use simple_error;
use simple_logger;
```
This will generate a clippy warning: https://rust-lang.github.io/rust-clippy/master/#single_component_path_imports